src/serverless-functions/send-to-phone.ts

```diff
@@ -0,0 +1,43 @@
+import "source-map-support/register";
+
+import { EFNYC_HOST, serverlessRollbarHandler } from "../utils/serverless-util";
+import { Twilio } from "twilio";
+
+export const handler = serverlessRollbarHandler(async (event) => {
+  const {
+    TWILIO_ACCOUNT_SID,
+    TWILIO_AUTH_TOKEN,
+    TWILIO_PHONE_NUMBER,
+  } = process.env;
+  if (!(TWILIO_ACCOUNT_SID && TWILIO_AUTH_TOKEN && TWILIO_PHONE_NUMBER)) {
+    throw new Error("Twilio credentials are not in environment!");
+  }
+
+  const userPhone =
+    event.queryStringParameters && event.queryStringParameters.userPhone;
+
+  const resultsPagePath =
+    event.queryStringParameters && event.queryStringParameters.userPath;
+
+  if (!userPhone) {
+    return {
+      statusCode: 400,
+      body: "No receiving phone number given!",
+    };
+  }
```

Comment:
It might be a good idea to validate the phone number somewhat here, just to be safe.  I don't think anything especially horrible will happen if we give twilio a totally crazy phone number here but it's good to follow the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle).

Reply:
Yep, good point.